docs/guides/functions.ipynb

```diff
@@ -440,7 +414,33 @@
   },
   {
    "cell_type": "markdown",
-   "id": "9137071a-e39c-4b66-afbd-697db38cc670",
+   "id": "af7fa72a-f272-43ff-800d-5ae37dd42b2b",
+   "metadata": {},
+   "source": [
+    "To retrieve a job's job_id by its position in the list, use `catalog.get_job_by_id()`:"
```

Comment:
@johannesgreiner this seems wrong to me to say "by its position in the list"? I think this should instead read as something like:

> If you already have the job ID for a certain job, you can retrieve it with `catalog.get_job_by_id()`:

Reply:
Isn't the job id the thing you're retrieving, and rather you're trying to find the nth job id in a list?

Reply:
No, `get_job_by_id` is when you already have a particular job ID (some string) and you are getting a job (I think https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/runtime-job-v2?)

> and rather you're trying to find the nth job id in a list?

That's what you're doing with `old_jobs[0]`: you're getting the nth job where `old_jobs` is all the jobs that have run, sorted by date.